src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,15 @@
-export const UserInfo = () => {};
+import { FC } from 'react';
+import { User } from '../types';
+
+type Props = {
+  user: User | undefined;
+};
+export const UserInfo: FC<Props> = ({ user }) => {
+  return (
+    user && (
+      <a className="UserInfo" href={`mailto:${user.email}`}>
+        {user.name}
+      </a>
```

Comment:
this check should be in UserList, when user isn't defined we shouldn't even render UserInfo